cetmix_tower_server/tests/test_plan.py

```diff
@@ -601,3 +601,67 @@ def test_plan_with_action_variables_for_condition(self):
             new_plan_log_records.command_log_ids[1].is_skipped,
             msg="The second plan line should not be skipped",
         )
+
+    def test_flight_plan_copy(self):
+        """Test duplicating a Flight Plan with lines and actions"""
+
+        # Create a Flight Plan
+        plan = self.Plan.create(
+            {
+                "name": "Test Flight Plan",
+                "note": "Test Note",
+            }
+        )
+
+        # Create a command for the plan line
+        command = self.Command.create(
+            {
+                "name": "Test Command",
+                "code": 'echo "Test"',
+            }
+        )
+
+        # Create a Flight Plan Line
+        plan_line = self.plan_line.create(
+            {
+                "plan_id": plan.id,
+                "command_id": command.id,
+                "path": "/test/path",
+            }
+        )
+
+        # Create an Action for the Plan Line
+        self.plan_line_action.create(
+            {
+                "line_id": plan_line.id,
+                "action": "n",  # next action
+                "condition": "==",
+                "value_char": "0",  # condition for success
+            }
+        )
+
+        # Duplicate the Flight Plan
+        copied_plan = plan.copy()
+
+        # Ensure the new Flight Plan was created with a new ID
+        self.assertNotEqual(copied_plan.id, plan.id)
+
+        # Check that the copied plan has the same number of lines
+        self.assertEqual(len(copied_plan.line_ids), len(plan.line_ids))
+
+        # Check that the copied plan's lines have the same actions as the original
+        original_line = plan.line_ids[0]
+        copied_line = copied_plan.line_ids[0]
```

Comment:
We need to ensure that:

- command
- condition
- custom path

are the same in the copied flight plan line

Reply:
Thanks, I'll fix it